src/components/Modals/PublishCollectionModal/PublishCollectionModal.tsx

```diff
@@ -57,6 +58,8 @@ export default class PublishCollectionModal extends React.PureComponent<Props, S
       totalPrice += rarityPrice
     }
 
+    const hasInsufficientMANA = !!wallet && wallet.networks.MATIC.mana <= 0
```

Comment:
I think that we need to also compare if the user has at least the amount of mana necessarily to publish the collection